### DISCUSSION TITLE
Trivia

### DISCUSSION THREAD

**Discussion by danielsauceda** [Supinic/supibot#24] [Ideas]
One thing we really like in offline chat is trivia and hangman. these are in the old ! based bot. If there were more games, questions  or something that were in the supiBot that would be awesome

**Comment by Supinic** [comment]
This could technically be implemented  via a command + chat module combination. The command would turn on the module, and then that would listen in to user input for hangmen, or answers for trivia.

However, I don't feel like this feature would be worth implementing in Supibot, mostly because there are other bots specializing in this functionality. Trivias and hangmen in chat isn't really a novel concept.
